nixos/modules/programs/firejail.nix

```diff
@@ -39,9 +44,15 @@ in {
   };
 
   config = mkIf cfg.enable {
-    security.wrappers.firejail.source = "${lib.getBin pkgs.firejail}/bin/firejail";
+    security.wrappers.firejail.source = "${lib.getBin fj}/bin/firejail";
 
-    environment.systemPackages = [ wrappedBins ];
+    environment.systemPackages = [ wrappedBins ] ++ optional cfg.firecfg fj;
+
+    environment.extraSetup = optionalString cfg.firecfg ''
+      mkdir tmp
+      cp -r $out tmp
+      chroot tmp firecfg
```

Comment:
I don't think chroot could ever work in a Nix builder as it requires root. You might be able to use unshare, however: http://man7.org/linux/man-pages/man1/unshare.1.html

In this cast though, I think it might be best to patch firecfg to support a ```--prefix``` option that it can look in instead of ```/usr```.

Reply:
Yes,it does have a `--bindir` option that does exactly that.
However,now it expects a username(see https://github.com/netblue30/firejail/blob/5e09cfb4f901d944c0418fcb041d4e96448028a2/src/firecfg/main.c#L286 )
How do I get the name of the current builder?